public/script.js

```diff
@@ -1,79 +1,100 @@
 
 /*
  * Emoji cheat sheet
  */
+$(document).ready(function() {
+  try {
+    if(document.flashtest && document.flashtest.PercentLoaded()>=0){
+      // Flash was able to load the test swf and is working
+      initZeroClipboard();
+    } else {
+      initJsClipboard();
+    }
+  } catch (e) {
+    initJsClipboard();
+  }
 
-$(function() {
-  if (FlashDetect.installed) {
-    var reglue;
-    var clip_flash_block_detect = new ZeroClipboard.Client();
-    clip_flash_block_detect.glue(document.getElementById('flash-test'));
-
-    var clip = new ZeroClipboard.Client();
-    clip.reposition(document.getElementById('flash-test'));
-    clip.receiveEvent('mouseover', null);
-
-    $.jnotify.setup({ 'delay': 1000, 'fadeSpeed': 500 });
+  function initZeroClipboard(){
+    ZeroClipboard.config({ 
+      forceHandCursor: true,
+      hoverClass: "hover"
+    });
+    var clipboardclient = new ZeroClipboard();
+
+    clipboardclient.on('ready', function( readyEvent ) {
+      $('ul').on('mouseover', 'div', function() {
+        try {
+          clipboardclient.clip(this);
+        } catch(e) {
+          console.log(e);
```

Comment:
which will not benefit anyone (except when working on it yourselves.

Pushing an event to GA might be better as it would report them to the maintainer